packages/neon-core/src/sc/ScriptBuilder.ts

```diff
@@ -1,23 +1,30 @@
 import BN from "bn.js";
 import {
   ensureHex,
-  int2hex,
   num2hexstring,
   reverseHex,
   str2hexstring,
-  StringStream
+  StringStream,
+  Fixed8
 } from "../u";
 import ContractParam, {
   ContractParamType,
   likeContractParam
 } from "./ContractParam";
 import OpCode from "./OpCode";
+import InteropServiceCode from "./InteropServiceCode";
+import { OpCodePrices } from "./OpCodePrices";
+import { getInteropServicePrice } from "./InteropServicePrices";
 
 export interface ScriptIntent {
-  scriptHash: string;
+  scriptHash: string | "NEO" | "GAS" | "POLICY";

```

Comment:
these are all strings too.

Reply:
I add these because I want to user to see that they can use these stings for convenient.